templates/account.html.twig

```diff
@@ -36,7 +36,7 @@
 "repositories": [
     {
         "type": "composer",
-        "url": "https://packages.friendsofshopware.com/",
+        "url": "{{ url('home') }}",
```

Comment:
Can you use APP_URL the env?

Reply:
Isn't APP_URL affecting how the home route is structured?

Reply:
Nope

Reply:
But it works out for me like this :( but I am fine with using the env as well. Do I have to make sure it has a trailing slash?